.jenkins-scripts/test-ceph.sh

```diff
@@ -6,7 +6,7 @@ source .jenkins-scripts/jenkins-common.sh
 cd "${ROOT_DIR}" || exit 1
 
 # Deploy rook, fail if it takes longer than 5 minutes
-timeout 300 ./scripts/k8s_deploy_rook.sh
+timeout 300 ./scripts/k8s/deploy_rook.sh
 
 # Poll for completion, fail if it takes longer than 30 minutes
-timeout 1800 ./scripts/ceph_poll.sh
```

Comment:
We should probably rename this while we are at it, so that we have deploy_x, verify_x, poll_x, delete_x.